### PR TITLE
Scrollbar in Project Description

### LINKED ISSUE #1238: Large empty space(left side), if Description of the project is too large
**Describe the bug**

If the circuit description is too large, then the page has lots of white space (Desktop View) below the circuit. 

**To Reproduce**
1. Go to 'https://circuitverse.org/users/3464/projects/16521' (Desktop View)
2. Scroll down
3. See Large empty space on left side (Desktop View)

**Expected behavior**

**Screenshots**

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by akansha-singh**
Fixes #1238 

#### Describe the changes you have made in this PR -
Added a Verticle Scrollbar in Project description div

### Screenshots of the changes (If any) -
ezgif com-video-to-gif(5)

### CODE REVIEW

**Comment by akansha-singh** [comment]
> @akansha-singh align the description to the Left as in the older version. Here in your PR the description is slightly indented towards the right. Please fix that +1.

WhatsApp Image 2020-03-20 at 17 00 34

**Comment by akansha-singh** [comment]
@nitin10s please review.

File: app/assets/stylesheets/logixRelease.scss [hunk 1]
```diff
@@ -226,6 +226,14 @@ a.btn.btn-outline-danger {
   text-decoration: none;
 }
 
+.scroll {
+  height: 180px;
+  margin-right: 4px;
+  overflow-y: auto;
+  padding: 4px;
+  text-align: justify;
```
Comment (nitin10s) [inline app/assets/stylesheets/logixRelease.scss]: text align: left , might be better?

Comment (akansha-singh) [reply app/assets/stylesheets/logixRelease.scss]: WhatsApp Image 2020-03-20 at 20 21 05

**Comment by nitin10s** [comment]
Can you please also decrease the margin b/w heading "Description" and the desc text.

**Comment by akansha-singh** [comment]
> Can you please also decrease the margin b/w heading "Description" and the desc text.

WhatsApp Image 2020-03-20 at 21 57 44

**Pull request merged**